provisioning/README.md

```diff
@@ -0,0 +1,370 @@
+
+```console
+com.dremio.jdbc.Driver-metadataProvider = it.unibz.inf.ontop.dbschema.impl.KeyAwareDremioDBMetadataProvider
+com.dremio.jdbc.Driver-schemas = HI_TEST_OEM, TRACE_TEST_OEM
+com.dremio.jdbc.Driver-tables.HI_TEST_OEM = CX_RUL_SerialPartTypization_Vehicle,CX_RUL_SerialPartTypization_Component,CX_RUL_AssemblyPartRelationship,CX_RUL_LoadCollective
+com.dremio.jdbc.Driver-unique.HI_TEST_OEM.CX_RUL_SerialPartTypization_Vehicle = UC_VEHICLE
+com.dremio.jdbc.Driver-unique.HI_TEST_OEM.CX_RUL_SerialPartTypization_Component = UC_COMPONENT
+com.dremio.jdbc.Driver-unique.HI_TEST_OEM.CX_RUL_AssemblyPartRelationship = UC_ASSEMBLY
+... 
+```
+* the ability to optimize to a specific virtualization engine although using a generic REST-based JDBC driver from Apache Calcite
+
+```console
+# Use the Data Virtualization backend
+jdbc.url=jdbc\:avatica\:remote\:url=http://data-backend:8888/druid/v2/sql/avatica/
+jdbc.driver=org.apache.calcite.avatica.remote.Driver
+org.apache.calcite.avatica.remote.Driver-metadataProvider = it.unibz.inf.ontop.dbschema.impl.DruidMetadataProvider
+org.apache.calcite.avatica.remote.Driver-typeFactory = it.unibz.inf.ontop.model.type.impl.DefaultSQLDBTypeFactory
+org.apache.calcite.avatica.remote.Driver-symbolFactory = it.unibz.inf.ontop.model.term.functionsymbol.db.impl.DefaultSQLDBFunctionSymbolFactory 
+```
+
+### Security
+
+Besides the authentication of the Ontop engine at the relational database via jdbc (one url/user per endpoint), there are no 
+additional (row-level) security mechanism.
+
+Hence we recommend to apply a role-based approach.
+
+For any accessing role:
+- define a separate database schema with appropriately filtered views
+- define a separate endpoint/port/mapping.
+
+### Data Sources and Scaleablility
+
+For the sample deployments, we use single agent container with an embedded database (H2) and/or a second database virtualization container (Dremio Community Edition) using preloaded files.
+
+Practical deployments will 
+* scale and balance the agent containers (for which the lifecycle hooks are already provided).
+* use an enterprise-level database (Postgres Service) or database virtualization infrastructure (Dremio Enterprise, Denodo, Teii) that are backed by an appropriate storage system (ADSL, S3, Netapp).
+
+## Deployment
+
+### Compile, Test & Package
+
+```console
+mvn package
+```
+
+This will generate
+- a [pluging jar](target/provisioning-agent-1.9.5-SNAPSHOT.jar) which maybe dropped into an Ontop server (into the lib folder)
+
+### Containerizing (Provisioning Agent)
+
+You could either call
+
+```console
+mvn install -Pwith-docker-image
+```
+
+or invoke the following docker command after a successful package run
+
+```console
+docker build -t ghcr.io/eclipse-tractusx/knowledge-agents/provisioning-agent:1.9.5-SNAPSHOT -f src/main/docker/Dockerfile .
```

Comment:
Do not use ghcr.io but Docker Hub. Please refer to [TRG 4.05 - Container registries](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05).

This comment addresses all appearances of ghcr.io.

Reply:
moved to dockerhub/tractusx